recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
   host:
     - icu       # [not win]
     - libnsl  # [linux]
+    - libcurl
```

Comment:
I think this also needs to be added to the `run` section...

Reply:
And both `host` and `run` need to be `[not win]`?

Reply:
Thanks! Updated!

Reply:
Can you fix this linting error by inserting another space before the `#`